genesis/process/argGenesisBlockCreator.go

```diff
@@ -35,6 +35,7 @@ type coreComponentsHandler interface {
 	TxVersionChecker() process.TxVersionCheckerHandler
 	ChainID() string
 	EnableEpochsHandler() common.EnableEpochsHandler
+	Rater() sharding.PeerAccountListAndRatingHandler
```

Comment:
Also for `Rater` there's no check for nil in `genesisBlockCreator.go`

Reply:
Added check + test